assets/css/index.css

```diff
@@ -0,0 +1,323 @@
+  font-weight: 700;
+  line-height: 8rem;
+}
+
+.slogan-gradient {
+  background-image: linear-gradient(119deg, #6d6afe 0%, #ff9f9f 100%);
+}
+
+.background-clip-text {
+  background-clip: text;
+  -webkit-background-clip: text;
+  color: transparent;
+}
+
+.hero-image {
+  width: 111.8rem;
+  height: 65.9rem;
+}
+
+section {
+  display: grid;
+  justify-content: center;
+  column-gap: 15.7rem;
+  row-gap: 1rem;
+  width: 100%;
+  padding: 5rem 0;
+}
+
+section:nth-of-type(odd) {
+  grid-template:
+    '. image'
+    'title image'
+    'description image'
+    '. image'
+    /29.1rem 55rem;
+}
+
+section:nth-of-type(even) {
+  grid-template:
+    'image .'
+    'image title'
+    'image description'
+    'image .'
+    /55rem 29.1rem;
+}
+
+.title {
+  grid-area: title;
+  font-size: 4.8rem;
+  font-weight: 700;
+  line-height: normal;
+  letter-spacing: -0.03rem;
+}
+
+.title-1-gradient {
+  background-image: linear-gradient(117deg, #fe8a8a 2.29%, #a4ceff 100%);
+}
+.title-2-gradient {
+  background-image: linear-gradient(304deg, #6fbaff 0%, #ffd88b 100%);
+}
+.title-3-gradient {
+  background-image: linear-gradient(133deg, #2945c7 0%, #dbe1f8 100%);
+}
+.title-4-gradient {
+  background-image: linear-gradient(310deg, #fe578f 0%, #68e8f9 100%);
+}
+
+.description {
+  grid-area: description;
+  font-family: Abel;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #6b6b6b;
+  line-height: 150%;
+}
+
+.content-image {
+  grid-area: image;
+  width: 55rem;
+  height: 45rem;
+}
+
+footer {
+  width: 100%;
+  height: 28rem;
+  padding-top: 3.2rem;
+  background-color: #111322;
+}
+
+.footer-box {
+  display: grid;
+  grid-template:
+    'copyright footer-links sns'
+    /1fr 1fr 1fr;
+  justify-items: center;
+  width: 100%;
+  max-width: 192rem;
+}
+
+.copyright {
+  grid-area: copyright;
+  color: #676767;
+  font-family: Arial;
+  font-size: 1.6rem;
+}
+
+.footer-links {
+  grid-area: footer-links;
+  display: flex;
+  column-gap: 3rem;
+  padding-right: 1.8rem;
+}
+
+.footer-link {
+  color: #cfcfcf;
+  font-family: Arial;
+  font-size: 1.6rem;
+}
+
+.sns {
+  grid-area: sns;
+  display: flex;
+  column-gap: 1.2rem;
+  height: 2rem;
+}
+
+/* tablet */
+@media screen AND (min-width: 768px) AND (max-width: 1199px) {
+  nav {
+    justify-content: space-evenly;
+    padding: 0 3.2rem;
+  }
+  nav a:first-child {
+    margin-right: 43rem;
+  }
+
+  nav a {
+    flex-shrink: 0;
+  }
+
+  .slogan {
+    font-size: 6.4rem;
+    line-height: 8rem;
+  }
+
+  .hero-header img {
+    width: 65rem;
+    height: 38.3rem;
+  }
+
+  section {
+    height: 26rem;
+    padding: 5rem 25rem;
+    column-gap: 5.1rem;
+    row-gap: 0.1rem;
+  }
+
+  section:nth-of-type(odd) {
+    grid-template:
+      '. image'
+      'title image'
+      'description image'
+      '. image'
+      /19rem 291px;
+  }
+
+  section:nth-of-type(even) {
+    grid-template:
+      'image .'
+      'image title'
+      'image description'
+      'image .'
+      /291px 19rem;
+  }
```

Comment:
레이아웃이 요구사항과 달리 우측이 옆으로 넘칩니다.

<img width="430" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/20a47822-38fe-47be-b51d-393ae211b419">
<img width="630" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/b4e8a3d8-391b-4649-89ac-99b02fb4edf4">

그리드 내 좌측 아이템의 너비와 우측 아이템 너비를 다시 확인해주세요. 
제 생각에는 padding은 빼고 유동적으로 변할 수 있는 부분은 auto 를 주는 게 나을 것 같습니다. 참고만 해보시고 좋은 방법 찾으셔서 반영 부탁드려요 😄 

```suggestion
  section {
    height: 26rem;
    column-gap: 5.1rem;
    row-gap: 0.1rem;
  }

  section:nth-of-type(odd) {
    grid-template:
      '. image'
      'title image'
      'description image'
      '. image'
      / auto 262px;
  }

  section:nth-of-type(even) {
    grid-template:
      'image .'
      'image title'
      'image description'
      'image .'
      / 262px auto;
  }
```

Reply:
일단 해당 부분은 제가 크기를 잘못 작성하여 나온 이슈로 확인되었습니다.

px로 다시 정의해보려다가 다른 단위를 한번 사용해볼까 하여 1fr (fr 단위는 그리드 컨테이너에 남아 있는 사용 가능한 공간의 일정 비율을 나타냅니다 [출처](https://developer.mozilla.org/ko/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout)

정확한 이유는 아직 파악하지 못했지만 auto로 하는 경우 .title, .description이 화면 끝까지 늘어나 현재 0.5fr로 수정하였습니다.

아직 레이아웃을 작성할때 쓰는 단위에 대해서 고민하고 있습니다.